decider-bcl2fastq/src/main/java/ca/on/oicr/pde/deciders/Bcl2fastqDeciderCli.java

```diff
@@ -351,11 +360,12 @@ public ReturnValue do_run() {
             decider.run();
         } catch (Exception e) {
             log.error(e);
+            e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1148)